aws/resource_aws_elastic_beanstalk_environment.go

```diff
@@ -496,6 +562,10 @@ func resourceAwsElasticBeanstalkEnvironmentRead(d *schema.ResourceData, meta int
 		return err
 	}
 
+	if err := d.Set("arn", env.EnvironmentArn); err != nil {
```

Comment:
Nitpick: Unfortunately the code around it does not follow the convention, but for simple `string` attributes, we do not need the `err` handling logic.

```go
d.Set("arn", env.EnvironmentArn)
```